CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ New Features
 - Adds a new utility function ``desc.compat.contract_equilibrium`` which takes in an ``Equilibrium`` object and an argument ``inner_rho``, and returns a new ``Equilibrium`` with original ``Equilibrium``'s ``inner_rho`` flux surface as its boundary.
 Optionally can also contract the profiles of the original ``Equilibrium`` so that the new ``Equilibrium``'s profiles match the original's in real space.
 - Adds second-order NAE constraints, accessible by passing ``order=2`` to ``desc.objectives.get_NAE_constraints``.
+- Adds `desc.plotting.plot_field_lines` function, which integrates and then plots magnetic field line trajectories in 3D.
+- `desc.magnetic_fields.field_line_integrate` now returns the `phi` values where the `r` and `z` values belong to. Although the documentation was mentioning that the output can be in negative `phis`, it was not clear and open to misinterpretation.
```

Comment:
I think this is now outdated, but should mention that `field_line_integrate` now outputs at the correct phi for negative fields